resources/user.py

```diff
@@ -149,31 +149,31 @@ def post(self, user_id):
 
 class UserLogin(Resource):
     parser = reqparse.RequestParser()
-    parser.add_argument('email',type=str,required=True,help="This field cannot be blank.")
-    parser.add_argument('password', type=str, required=True, help="This field cannot be blank.")
+    parser.add_argument('email',type=str,required=True,help="This field cannot be blank")
+    parser.add_argument('password', type=str, required=True, help="This field cannot be blank")
 
     def post(self):
         data = UserLogin.parser.parse_args()
 
         user = UserModel.find_by_email(data['email'])
 
         if user and user.archived:
-            return {"message": "Not a valid user"}, 403
+            return {"message": "Invalid user"}, 403
```

Comment:
Note. I think if a user is archived then perhaps a "User not found" response is more appropriate. Simply making a note of this. (No action needed)